lib-multisrc/machinetranslations/src/eu/kanade/tachiyomi/multisrc/machinetranslations/MachineTranslations.kt

```diff
@@ -203,9 +230,49 @@ abstract class MachineTranslations(
         return FilterList(filters)
     }
 
+    override fun setupPreferenceScreen(screen: PreferenceScreen) {
+        // Some libreoffice font sizes
+        val sizes = arrayOf(
+            "24", "26", "28",
+            "32", "36", "40",
+            "42", "44", "48",
+            "54", "60", "72",
+            "80", "88", "96",
+        )
+
+        ListPreference(screen.context).apply {
+            key = FONT_SIZE_PREF
+            title = intl["font_size_title"]
+            entries = sizes.map {
+                "${it}pt" + if (it == DEFAULT_FONT_SIZE) " - ${intl["default_font_size"]}" else ""
+            }.toTypedArray()
+            entryValues = sizes
+            summary = intl["font_size_summary"]
+
+            setOnPreferenceChangeListener { _, newValue ->
+                val selected = newValue as String
+                val index = this.findIndexOfValue(selected)
+                val entry = entries[index] as String
+
+                fontSize = selected.toInt()
+
+                Toast.makeText(
+                    screen.context,
+                    intl["font_size_message"].format(entry),
+                    Toast.LENGTH_LONG,
+                ).show()
+
+                true
```

Comment:
```suggestion
                false
```
since your `fontSize` var already saves the pref